### ISSUE TITLE
Alamo Website: Non-tracking cookie blocked, causing the missing images

### ISSUE THREAD

**Issue by nokisobar** [EFForg/privacybadger#2432]
#### What is your browser and browser version?

Chrome latest official version on Windows 7 Professional Edition (desktop)

#### What is broken and where?

Our Alamo website is currently missing the service-driven vehicle images because a non-tracking cookie is blocked by Privacy Badger at *assets-xqa.nonprod.gcs.ehi.com*. This domain is used for the enterprise-level Adobe Experience Manager (AEM) which is a Content Management System (CMS) that serves the image assets and content to our Alamo web client application. 

**Issue:** The vehicle image is missing now.

![QA_5331 Aug  14 23 10](https://user-images.githubusercontent.com/53952688/63070944-293d8380-beeb-11e9-9603-d6744e4d36c0.jpg)

**Expected Result:** We should see the vehicle image appearing on the website as shown below.

![QA_5332 Aug  14 23 15](https://user-images.githubusercontent.com/53952688/63070707-4faeef00-beea-11e9-86d3-84f8f1cbaf15.jpg)

#### What is the "culprit" domain?

assets-xqa.nonprod.gcs.ehi.com

#### What is your debug output for this domain?

```
**** ACTION_MAP for ehi.com
assets-xqa.nonprod.gcs.ehi.com {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1566296098120,
  "userAction": ""
}
assets.gcs.ehi.com {
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1566204600365,
  "userAction": ""
}
ehi.com {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 0,
  "userAction": ""
}
static.ehi.com {
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1566111848513,
  "userAction": ""
}
**** SNITCH_MAP for ehi.com
ehi.com [
  "ehiaws.com",
  "nationalcar.com",
  "alamo-np.ie"
]
```
Please don't hesitate to reach out to me at any time if further information is needed to resolve this issue for us. Thanks in advance!

**Comment by ghostwords** [comment]
We'll address this together with #2430.

**Comment by nokisobar** [comment]
@ghostwords  Hold on please! #2340 has a totally different culprit domain and it's not the same as assets-xqa.nonprod.gcs.ehi.com. Please leave this ticket open. Thanks!

**Comment by nokisobar** [comment]
The culprit domain of #2432 (which is this ticket) is assets-xqa.nonprod.gcs.ehi.com.
The culprit domain of #2430 (which is another ticket) is *.location.enterprise.com.

They are two different domains.

**Comment by ghostwords** [comment]
Sure, but the underlying issue is the same. Let's continue in #2340.
